test/buildstrategy_samples.go

```diff
@@ -21,10 +21,12 @@ spec:
       image: quay.io/containers/buildah:v1.31.0
       workingDir: $(params.shp-source-root)
       securityContext:
-        privileged: true
+        capabilities:
+          add: ["SETFCAP"]
       command:
         - /usr/bin/buildah
       args:
+        - --storage-driver=$(params.storage-driver)
```

Comment:
@SaschaSchwarze0 If I specify the parameter in the test examples, the taskrun expects 7 system parameters here instead of 6. Given that `storage-driver` is being specified in the buildstrategy & user configurable, I don't think it fits in the system parameters category. (I do not see any other parameters like `build-args` etc in this category)
So, I am not sure if adding the parameter is solving the actual problem. Please let me know in case I am misinterpreting.